demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -736,7 +737,7 @@ def get_master_private_repo_meta_file(self, metadata_file_path: str):
                 click.secho("Running on master branch - skipping price change validation", fg="yellow")
             return None
         try:
-            old_meta_file_content = current_repo.git.show(f'{self.prev_ver}:{metadata_file_path}')
+            old_meta_file_content = current_repo.git.show(f'{self.prev_ver}:{relative_file_path}')
```

Comment:
Let's be consistent here and get the file the same way we do everywhere else
```suggestion
            old_meta_file_content = get_remote_file(metadata_file_path, tag=self.prev_ver)
```